util/fbrun/FbRun.cc

```diff
@@ -418,12 +418,12 @@ void FbRun::tabComplete(const std::vector<std::string> &list, int &currentItem,
         currentItem = 0;
     else if (currentItem >= list.size())
         currentItem = list.size() - 1;
-    int item = currentItem;
+    unsigned int item = currentItem;
 
     while (true) {
         if (reverse) {
-            if (--item < 0)
-                item = list.size() - 1;
+	  if (item == 0) item = list.size() - 1;
```

Comment:
except if you print source code like i do :)

but i agree.

Reply:
Apologies for the missing newline; I had been working on another project with the style that if there were no braces arond the then clause, it was _required_ to be on the same line as the condition, and inadvertently did that here. In any case, moot now because the warning mitigation has been separated.